docker/Dockerfile-dev

```diff
@@ -4,7 +4,10 @@
 
 FROM python:3.5-alpine
 
+RUN apk --update --no-cache add \
+	sqlite > /dev/null 2>&1
 ADD requirements.txt /requirements.txt
+ADD docker/db/schema.db /sqlite.db
```

Comment:
This is not ready for production - I guess it would rather link the directory. Also manager is needed to run create_all on an empty database.